README.md

```diff
@@ -89,32 +89,34 @@ Next, clone this repository into a parent folder:
 
 ```
 (figshare_patrons) $ cd /path/to/parent/folder
-(figshare_patrons) $ git clone https://github.com/UAL-RE/ReQUIAM.git
+(figshare_patrons) $ git clone https://github.com/UAL-RE/ReQUIAM.git   (this is to clone a copy of the project to local directory) 
 ```
 
 With the activated `conda` environment, you can install with the `setup.py` script:
 
 ```
 (figshare_patrons) $ cd /path/to/parent/folder/ReQUIAM
-(figshare_patrons) $ (sudo) python setup.py develop
+(figshare_patrons) $ python setup.py develop (this is to setup all the necessary packages for this package. Without them, you cannot run. if you see errors, check python PATH. You may consider to add "psudo() { sudo env PATH="$PATH" "$@"; }"  to .bashrc)
```

Comment:
Recommend keeping docs clean and avoiding redundant info (e.g., any dev should know that setup.py is supposed to install all the dependencies.). Also, I recommend explaining why and when one might want to use the psudo function you describe